src/Microsoft.VisualStudio.AppDesigner/PropPages/PropPage.vb

```diff
@@ -422,7 +416,7 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
             ' we need to adjust the size of the page if it's autosize or if we're native (in which
             ' case we're going to adjust the size of our secret usercontrol instead) See the Create
             ' for more info about the panel
-            If _propPage IsNot Nothing AndAlso pRect IsNot Nothing AndAlso pRect.Length <> 0 AndAlso (_propPage.AutoSize OrElse _hostedInNative) Then
+            If _propPage IsNot Nothing AndAlso pRect IsNot Nothing AndAlso pRect.Length <> 0 Then
```

Comment:
Yes, the condition was: `AndAlso (_propPage.AutoSize OrElse _hostedInNative)`, the code-path which used `SetParent` always set `_hostedInNative = True`, so the additional clause was no longer required and all code that used `_hostedInNative` could become the default.